### DISCUSSION TITLE
CLI Domain Alternative?

### DISCUSSION THREAD

**Discussion by CaptainPalapa** [Mailu/Mailu#3344] [User support] [answered]
First, let me say: great job on the documentation.

My question:  Is there a way, via the CLI, to add an "alternative" domain alias?  I know there's the config import that appears to handle it, but I can't determine if there's a way via the simple command line.

Create a domain (exists):
`docker compose exec admin flask mailu domain example.com`
...but not (that I can find) to create a domain alias (or "alternative"), maybe like:
`docker compose exec admin flask mailu alternative example.com altexample.com`

**Comment by ghostwheel42** [comment, marked as answer]
Hi,

you could use curl and the REST-API, ur you could use the cli command `config-import` in the _update_ mode like this:

```
$ docker compose exec -T admin flask mailu config-import -u <<<'{"domain":[{"name":"foo.com","alternatives":["bar.com"]}]}'
```

You can add `-nv` to show what would happen without changing the database.

**Warning**: If you run this command without the `-u` on a production database everything will get wiped and replaced by only `foo.com` and it's alternative.

**Reply by CaptainPalapa** [reply]
Great information, perfect, thank you!
